### PR TITLE
Get distro info from /etc/os-release from Debian distro.

### PR DESCRIPTION
**Pull request by LiliDeng**
By default, lsb_release is not installed in Debian but installed in Ubuntu.
Get release version from /etc/debian_version when it is debian distro.

### CODE REVIEW

File: lisa/operating_system.py [hunk 1]
```diff
@@ -330,7 +330,16 @@ def _get_information(self) -> OsInformation:
             if os_release_info.group("name") == "NAME":
                 vendor = os_release_info.group("value")
             elif os_release_info.group("name") == "VERSION_ID":
-                release = os_release_info.group("value")
+                # get release from /etc/debian_version for debian distro
+                # e.g. credativ debian 9-backports 9.20190313.0
+                # cat /etc/debian_version - 9.8
+                # VERSION_ID="9"
+                if "Debian" in vendor:
```
Comment (squirrelsc) [inline lisa/operating_system.py]: The logic should be in Debian, instead of posix.

Comment (LiliDeng) [reply lisa/operating_system.py]: Fixed

File: lisa/operating_system.py [hunk 2]
```diff
@@ -504,26 +500,40 @@ def _package_exists(self, package: str, signed: bool = True) -> bool:
...
-            cmd="lsb_release -a", shell=True, no_error_log=True
-        )
+        # try to set version info from /etc/os-release.
+        cat = self._node.tools[Cat]
+        cmd_result = cat.run("/etc/os-release")
         cmd_result.assert_exit_code(message="error on get os information")
-        assert cmd_result.stdout, "not found os information from 'lsb_release -a'"
 
+        vendor: str = ""
+        release: str = ""
```
Comment (sharsonia) [inline lisa/operating_system.py:509]: I don't think we need to initialize these to empty string. They are already initialized to empty string in the OsInformation object.

File: lisa/operating_system.py [hunk 3]
```diff
@@ -504,26 +500,40 @@ def _package_exists(self, package: str, signed: bool = True) -> bool:
         return False
 
     def _get_information(self) -> OsInformation:
-        cmd_result = self._node.execute(
-            cmd="lsb_release -a", shell=True, no_error_log=True
-        )
+        # try to set version info from /etc/os-release.
```
Comment (sharsonia) [inline lisa/operating_system.py:503]: wait, don't we already get the version and rest of the info from /etc/os-release in Posix class as well?
Why do we need it here?

Comment (LiliDeng) [reply lisa/operating_system.py:503]: the release is integer from /etc/os-release, e.g. debian debian-10 10-backports-gen2 0.20210201.535, release -> 10
we will see exception 'failed. LisaException: The version is invalid format: 10'

Then we get the release from /etc/debian_version for debian, that's why I overwrite it.

File: lisa/operating_system.py [hunk 4]
```diff
@@ -504,26 +500,40 @@ def _package_exists(self, package: str, signed: bool = True) -> bool:
...
+            elif os_release_info.group("name") == "VERSION":
+                codename = get_matched_str(
+                    os_release_info.group("value"),
+                    super()._distro_codename_pattern,
+                )
+            elif os_release_info.group("name") == "PRETTY_NAME":
+                full_version = os_release_info.group("value")
+
+        # get release from /etc/debian_version for debian distro
+        # credativ debian 9-backports 9.20190313.0
```
Comment (squirrelsc) [inline lisa/operating_system.py]: Add more explanation, so people know "credativ debian 9-backports 9.20190313.0" is a marketplace image name, and what's the raw output line.

**Review by squirrelsc — DISMISSED** [review]

File: lisa/operating_system.py (hunk 4 above)
Comment (LiliDeng) [reply lisa/operating_system.py]: update

**Review by squirrelsc — APPROVED** [review]

File: lisa/operating_system.py (hunk 3 above)
Comment (sharsonia) [reply lisa/operating_system.py:503]: I see. Can we somehow re-use the code snippet used in Posix class to get the os information from /etc/os-release and then just replace the version part from /etc/debian_version?

**Review by sharsonia — APPROVED** [review]

File: lisa/operating_system.py (hunk 3 above)
Comment (LiliDeng) [reply lisa/operating_system.py:503]: I'll refine it later.

**Pull request merged**